android-json-form-wizard/src/main/java/com/vijay/jsonwizard/interactors/JsonFormInteractor.java

```diff
@@ -44,7 +45,7 @@
 
     private static final String TAG = "JsonFormInteractor";
     private static final JsonFormInteractor INSTANCE = new JsonFormInteractor();
-    protected Map<String, FormWidgetFactory> map = new HashMap<>();
+    public static Map<String, FormWidgetFactory> map;
```

Comment:
should this be a static when its being modified in instance methods `registerWidgets()`

Reply:
Not sure what the issue would be in this case but I've removed the static modifier.